src/tests/hotreload-integration-test.ts

```diff
@@ -82,6 +88,85 @@ describe('Hot Code Reload for JS Particle', async () => {
     assert.deepEqual(slotConsumer.getRendering().model,  {name: 'Jack', age: '15'});
     assert.deepEqual(slotConsumer._content.template, `Hello <span>{{name}}</span>, new age: <span>{{age}}</span>`);
   });
+
+  it('ensures new handles are working', async () => {
```

Comment:
Just to make sure, we do not (currently) test that if the code changes, after the reload, the new code is in place -- is that correct? I realize this test is just checking that the handles rebuilt after a reload are still operational. Is it worth also, separately, testing code changes post-reload?

Reply:
No, this also tests that the new code is in place. The particles set the `age` field differently:

Before: `await util.assertSingletonWillChangeTo(arc, personStoreOut, 'age', 30);`
After: `await util.assertSingletonWillChangeTo(arc, personStoreOut, 'age', 18);`